docs/common/yum.md

```diff
@@ -121,7 +121,7 @@ The repositories to enable, as well as the instructions to enable them, are OS-d
 #### CentOS 8
 
 -   Ensure that the `extras` repo in `/etc/yum.repos.d/CentOS-Extras.repo` is enabled.
--   Ensure that the `PowerTools` repo in `/etc/yum.repos.d/CentOS-PowerTools.repo` is enabled.
+-   Ensure that the `powertools` repo in `/etc/yum.repos.d/CentOS-PowerTools.repo` is enabled.
```

Comment:
The repos also moved from Centos-foo to Centos-Linux-Foo. Probably easiest to have a "Centos 8.0 to 8.2" section and a "Centos 8.3 and up" section instead of putting them into bullet points.

Reply:
also a warning somewhere that if you enabled `PowerTools` before 8.3, then upgraded to 8.3, you will probably have to enable `powertools`.